yarn.lock

```diff
@@ -2005,12 +2007,16 @@ __metadata:
     style-dictionary: ^3.8.0
     stylelint: ^15.10.3
     stylelint-config-recommended: ^10.0.1
+    svg4everybody: ^2.1.9
     tailwindcss: ^3.3.3
     ts-jest: ^29.1.1
     typescript: ^4.9.5
   peerDependencies:
     react: ">= 17"
     react-dom: ">= 17"
+  bin:
+    eds-apply-theme: bin/eds-apply-theme.js
+    eds-init: bin/eds-init.js
```

Comment:
These are the commands consumers will have access to. e.g., 

`npx eds-init`
`npx eds-apply-theme`